### PR TITLE
refactor: Replaces compose_state.subject with compose_state.topic.

### LINKED ISSUE #9: Change exports.subject to exports.topic
In compose_state.js, there is the export object that many other classes reference. That has the variable subject that needs to be changed to topic.
See [UML](https://drive.google.com/file/d/1PHKKCSQjAl1t8DvzztCnEZxHF5XZn17U/view?usp=sharing)

Many classes will need to be changed (compose_actions.js etc) and it will most probably require many tests to be updated.

### PR DESCRIPTION
**Pull request by eywoon**
Changes compose_state class to include topic instead of subject.
All classes that reference compose_state.subject are changed into compose_state.topic.

Closes #9

### CODE REVIEW

File: frontend_tests/node_tests/Find Results [hunk 1]
```diff
@@ -0,0 +1,187 @@
+Searching 244 files for "compose_state.topic"
+
```
Comment (joakimboman) [inline frontend_tests/node_tests/Find Results]: This file should be deleted.

Comment (eywoon) [reply frontend_tests/node_tests/Find Results]: Good catch

**Review by emillind — APPROVED** [review]
This looks good to me. I will handle the merging when some other reviews drop in.

**Review by joakimboman — APPROVED** [review]
For some reason the tests did not work but now they do, and I have ran them multiple times. So all good! Might have been something on my end.

**Pull request merged**